indigo-frontend/src/core/components/common/form-dialog/form-dialog.component.html

```diff
@@ -64,6 +64,7 @@ <h2 class="text-lg font-semibold text-neutral-900">{{ title }}</h2>
             <eln-button
               [type]="'button'"
               [variant]="'blue'"
+              [disabled]="form.invalid"
```

Comment:
Verified the change across all known usages; observed no functional disruptions on my end